rush-plugins/rush-init-project-plugin/src/cli.ts

```diff
@@ -22,6 +22,11 @@ import { TerminalSingleton } from "./terminal";
       "Enable output verbose debug purposing messages",
       false
     )
+    .option(
+      "-u, --ui-mode",
```

Comment:
Could we remove `-u` and rename `--ui-mode` to `--ui`

The user story could be `rush init-project --ui`